pkg/views/apikey/notify.go

```diff
@@ -14,9 +14,12 @@ import (
 func Render(key, apiUrl string) {
 	var output string
 
-	output += fmt.Sprintf("%s %s", views.GetPropertyKey("Generated API key: "), key) + "\n\n"
+	output += "You have successfully generated a new API Key and Server URL, which you can find below:\n\n"
 
-	output += "You can connect to the Daytona Server from a client machine by running:"
+	output += fmt.Sprintf("%s %s", views.GetPropertyKey("Daytona_API_Key="), key) + "\n"
```

Comment:
`Daytona_API_Key` should be `DAYTONA_API_KEY`
`Daytona_SERVER_URL` should be `DAYTONA_SERVER_URL`.

Also, those 2 lines should be printed without indentation so they can be copied. Similarly to how the `command` variable is printend. (Just without the green styling)

Reply:
sure.

Reply:
sure.

Reply:
![Screenshot 2024-12-19 at 10 47 59](https://github.com/user-attachments/assets/3598dd5e-c1de-45f4-ad4c-56e51de5ca38)

@kartikaysaxena the order is wrong now. The env vars display should go under the `You have successfully generated a new API Key and Server URL, which you can find below` line

Reply:
![Screenshot 2024-12-19 at 5 09 38 PM](https://github.com/user-attachments/assets/59569630-e9fe-460f-81bc-1a40b3f28362)
@Tpuljak does this look fine? Updated the PR for the changes.